storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/LoggingInterceptor.java

```diff
@@ -0,0 +1,55 @@
+package com.pushtorefresh.storio.sqlite;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.pushtorefresh.storio.operations.PreparedOperation;
+
+import java.util.Locale;
+
+public class LoggingInterceptor implements Interceptor {
+
+    @NonNull
+    private final Logger logger;
+
+    public LoggingInterceptor(@NonNull Logger logger) {
```

Comment:
I suppose make it private and create via static method. To make it easer to support compatibility in the future)